projects/models.py

```diff
@@ -50,40 +35,18 @@ class Meta:
 class Project(models.Model):
     """
     Represents a startup project with details about funding, status, and documentation.
-
-    Validation:
-    - Current funding must not exceed the funding goal.
-    - Business plan is required if the project is in progress or completed.
-    - Funding goal is required if the project is marked as a participant.
-
-    Fields:
-    - startup: ForeignKey to the Startup that owns the project.
-    - title: Title of the project.
-    - description: Optional project description.
-    - business_plan: Optional uploaded business plan document.
-    - media_files: Optional uploaded media files related to the project.
-    - status: Project status with choices (draft, in progress, completed, cancelled).
-    - duration: Duration of the project in days.
-    - funding_goal: Target funding amount (optional).
-    - current_funding: Current amount of funding received.
-    - category: Category of the project.
-    - website: Project website URL (optional).
-    - email: Contact email for the project, must be unique.
-    - has_patents: Whether the project has patents.
-    - is_participant: Whether the project is a participant (e.g. in a program).
-    - is_active: Whether the project is active.
-    - created_at: Timestamp of creation.
-    - updated_at: Timestamp of last update.
     """
+
     STATUS_CHOICES = [
         ('draft', 'Draft'),
         ('in_progress', 'In Progress'),
         ('completed', 'Completed'),
         ('cancelled', 'Cancelled')
     ]
 
+    # Якщо модель Startup в apps.startups.models
     startup = models.ForeignKey(
-        'profiles.Startup',
+        'startups.Startup',  # або 'profiles.Startup' якщо модель там
```

Comment:
Good update to the new app location for Startup.
Remove the comment in production code or adjust it to clarify the migration path if still in transition.